html/css/index.css

```
@@ -1,31 +1,141 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&display=swap');
+
 html {
   /* Our life is easier if we don't have to think about default styling */
   all: unset;
+
+  /* Use same font for everything! */
+  font-family: "Montserrat", sans-serif;
+  font-optical-sizing: auto;
+  font-weight: normal;
+  font-style: normal;
+  font-size: 0.8rem;
 }
 
 body {
   background-color: black;
+  background-image: url("/images/loginbg.png");
+  background-size: cover;
   color: white;
-  font-family: sans-serif;
 }
```

This looks great!!!! I was wondering we can add `background-position: center 20%;` to have the bottles show a little more when the website is full screened but this is completely an opinion
```suggestion
body {
  background-color: black;
  background-image: url("/images/loginbg.png");
  background-size: cover;
  color: white;
  background-position: center 20%;
}
```

---

I didn't know this was a thing! Applied :)